scripts/scil_fodf_msmt.py

```diff
@@ -160,7 +160,16 @@ def main():
     if not is_normalized_bvecs(bvecs):
         logging.warning('Your b-vectors do not seem normalized...')
         bvecs = normalize_bvecs(bvecs)
-    gtab = gradient_table(bvals, bvecs, b0_threshold=b0_thr)
+
+    # Note. This script does not currently allow using a separate b0_threshold
+    # for the b0s. Using the tolerance. To change this, we would have to
+    # change many things in dipy. An issue has been added in dipy to
+    # ask them to clarify the usage of gtab.b0s_mask. See here:
+    #  https://github.com/dipy/dipy/issues/3015
+    # b0_threshold option in gradient_table probably unused.
+    args.b0_threshold = args.tolerance
```

Comment:
I see you did it here, checking the b0 with the tolerance. Might be easier with my proposition using the args. Also, why is there a `args.b0_threshold = `?

Reply:
Let's talk about it tomorrow. I changed the way to implement it a few times, this seemed best to me, but I'll explain.